source/community/about/community-guidelines.html.md

```diff
@@ -8,89 +8,43 @@ wiki_revision_count: 11
 
-### Search before you ask
+## Be respectful
 
-A few seconds on Google can help you find an answer to a question, or a past discussion on a subject, which can help us reduce duplication of effort and any associated frustration. Your effort will be appreciated.
+Our community is defined by the interactions of its members. When people are nice to each other, our community is nice. It can be easy to forget, in a disconnected world, that behind the IRC nickname or email address is a person. Please remember that when you communicate with other users, contributors, or anyone outside the community who interacts with us, whether in person or online.
 
-<form action="https://google.com/search" class="">
-<div class="row">
-<div class="col-xs-3">
-<select class="form-control" name="as_q">
-<option value="site:lists.ovirt.org OR (site:bugzilla.redhat.com ovirt)">Lists & bugzilla</option>
-<option value="site:lists.ovirt.org">Lists only</option>
-<option value="(site:bugzilla.redhat.com ovirt)">Bugzilla only</option>
-</select>
-</div>
-<div class="col-xs-6" style="padding:0">
-<input name="q" class="form-control">
-</div>
-<div class="col-xs-3">
-<button type="submit" class="btn btn-primary">Search Google</button>
-</div>
-</div>
-</form>
+Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It's important to remember that a community where people feel uncomfortable or threatened is not a productive one. 
 
-### Be pleasant
+## Be tolerant 
 
-Our community is defined by the interactions of its members. When people are nice to each other, our community is nice. It can be easy to forget, in a disconnected world, that the person behind the IRC nick is a person. Please remember that.
+We are a community of professionals, and we conduct ourselves professionally. We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
 
-For each of the communication channels the community uses, there are some channel-specific guidelines.
+Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
 
-## Mailing list guidelines
+* Violent threats or language directed against another person
+* Discriminatory jokes and language
+* Posting sexually explicit or violent material
+* Posting (or threatening to post) other people's personally identifying information ("doxing")
+* Personal insults, especially those using racist or sexist terms
+* Unwelcome sexual attention
```

Comment:
not entirely accurate, and this is a tricky subject, because the perception of the recipient is exactly what we're supposed to take into consideration. one could argue that they felt that the behavior of the recipient was interpreted as solicitation/invitation for this attention, thereby rolling the responsibility of "proof" to the recipient, when in fact the offender should have erred on the side of caution to begin with. I believe the word "unwelcome" is the correct word to use here.